robot-service/src/main/java/services/controllers/DefineUserType.java

```diff
@@ -0,0 +1,63 @@
+package services.controllers;
+
+import domain.CategoryName;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import services.model.response.ProfileResponse;
+
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Controller class. Manage controllers when user want to define his type.
+ */
+@RestController
+public class DefineUserType {
+
+    /**
+     *
+     * @param username
+     * @param password
+     * @return Response with books - this is because server define user profile base on statistics in db.
+     */
+    @RequestMapping(method = RequestMethod.POST, value = "/login")
+    public ProfileResponse login(@RequestParam (value = "username") String username,
+                                 @RequestParam (value = "password") String password) {
```

Comment:
:cactus: We know :) But encryptions, hashing, salting and other more complex things are not a part of objectives in this project.
Maybe, we will change this later if we will have enough time and of course Tomasz's approval :v: